department-of-veterans-affairs/abd-vro
# Feature - Connect CC API to Postgres

## What was the problem?


Associated tickets or Slack threads:
- https://github.com/department-of-veterans-affairs/vagov-claim-classification/issues/233

## How does this fix it?[^1]


## How to test this PR
- See [#Database Changes](https://github.com/department-of-veterans-affairs/abd-vro/blob/100cc030da1410854ba3e1051f6013e951e9c718/domain-cc/README.md#database-changes) in the domain-cc README.
- Make POST request to `http://localhost:8120/batch-classifier`. Ensure that claim + contention tables are updated.
- example POST request:
```
{
  "claim_id": 2,
  "form526_submission_id": 2,
  "contentions": [
    {
     "diagnostic_code": 6326
    },
    {
    "diagnostic_code": 6329
    }
  ]
}
```


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
